benchmarks/bench_async_vs_sync.py

```
@@ -0,0 +1,277 @@
+# ---------------------------------------------------------------------------
+
+def bench_sync_emit(n: int):
+    from ovos_bus_client.message import Message
+    bus = make_sync_bus()
+    msg = Message("benchmark.emit", {"payload": "x" * 128})
+
+    def do():
+        bus.emit(msg)
+
+    times = _timeit(do, n)
+    _stats(times, f"sync  emit ×{n}")
```

Ruff (0.15.5)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)